README.md

```diff
@@ -1,6 +1,6 @@
 Digital Marketplace GOV.UK Frontend ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
-[![Known Vulnerabilities](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend/badge.svg?targetFile=package.json)](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend?targetFile=package.json)
+[![Known Vulnerabilities](https://snyk.io/test/github/alphagov/digitalmarketplace-govuk-frontend/badge.svg?targetFile=package.json)](https://snyk.io/test/github/alphagov/digitalmarketplace-govuk-frontend?targetFile=package.json)
```

Comment:
Lovely 👍 I think the Snyk badges are broken in a bunch of our other repos too 🤦‍♀- if everyone keeps an eye out when they're doing a PR that'd be awesome

Reply:
I just double-checked and it looks like a bug with Snyk examples
![image](https://user-images.githubusercontent.com/3441519/66554567-25be2580-eb45-11e9-81f5-44cac523dd3b.png)

Reply:
I've sent them a message